click_odoo_contrib/initdb.py

```diff
@@ -35,7 +35,21 @@ def check_cache_prefix(cache_prefix):
             "Invalid cache prefix name '{}'".format(cache_prefix))
 
 
-def odoo_createdb(dbname, demo, module_names):
+def _patch_ir_attachment_store():
+
+    def _db_storage(self):
```

Comment:
@sbidoul I had to write it with old api to make it work on 9.0.

```python
@odoo.api.v7
def _db_storage(self, cr, uid, context=None):
    return 'db'
```

Reply:
The good thing is that the test suite caught it.